### ISSUE TITLE
Ensure Search and Go Input elements have a <label>

### ISSUE THREAD

**Issue by JulieVanilla** [vanilla/vanilla#3697]
**Client finding:** Labels MUST be programmatically associated with form input elements, using label (explicit or implicit), title, aria-labelled by or aria-label. Using a placeholder attribute as a substitute for a visible label for "Search" is contrary to the HTML5 specs. And also there is no label provided for the "Go" input field.

**Client Recommendation:** Ensure "Search" and "Go" input element has a <label>, since placeholder is contrary to the HTML5 specs.

**Comment by beckyvb** [comment]
There are 3 PRs to address this one: 
https://github.com/vanilla/locales/pull/10
https://github.com/vanilla/internal/pull/442
https://github.com/vanilla/vanilla/pull/3758

**Comment by beckyvb** [comment]
Resolved with: https://github.com/vanilla/vanilla/pull/3758
